README.md

```diff
@@ -141,13 +138,13 @@ merged into the RFC repository as a markdown file. At that point the RFC is
     if it is made prematurely.
     - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
       a *summary comment* trying to lay out the current state of the discussion
-      and major tradeoffs/points of disagreement.
+      and major trade-offs/points of disagreement.
     - Before actually entering FCP, *all* members of the subteam must sign off;
     this is often the point at which many subteam members first review the RFC
     in full depth.
   - The FCP lasts ten calendar days, so that it is open for at least 5 business
     days. It is also advertised widely,
-    e.g. in [This Week in Rust](https://this-week-in-rust.org/). This way all
+    e.g. in [Sawtooth Mailing List](hyperledger-stl@lists.hyperledger.org). This way all
```

Comment:
Links are inconsistent.  Some are hidden (the first and this one) and some are shown in the text (the second link, above). 

Also, I suggest using the full name for this mailing list: "[Hyperledger Sawtooth mailing list]"

P.S. Capitalizing "mailing list" makes me :cry: